orchagent/timer.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "selectable.h"
+#include "orch.h"
+
+namespace swss {
+
+class ExecutableTimer : public Executor
+{
+public:
+    ExecutableTimer(SelectableTimer *timer, Orch *orch)
+        : Executor(timer, orch)
+    {
+    }
+
+    SelectableTimer *getSelectableTimer()
```

Comment:
Noticed in rest of your code changes, the usage of "const" for class functions like this. Do you want to add it here as well?

Reply:
This wrapper class doesn't care about non const access to timer, so it will keep it as raw as possible. Do you find any harm for ExecutableTimer class?

---
In reply to: [150728600](https://github.com/Azure/sonic-swss/pull/382#discussion_r150728600) [](ancestors = 150728600)

Reply:
I don't see any harm. Just noticed the change between the two class defines